src/components/FeedPage/ListingFeed.js

```diff
@@ -1,25 +1,80 @@
-import React from 'react';
+import React, { useState } from 'react';
 import styled from 'styled-components';
 import ListingItem from './ListingItem';
 import PropTypes from 'prop-types';
+import Label from '../../misc/Label';
 
-const StyledListingFeed = styled.section`
+const StyledListingFeed = styled.section``;
+
+const StyledSearchBar = styled.div`
+  display: grid;
+  font-family: 'PT Mono', monospace;
+`;
+
+const StyledSelect = styled.select`
+  border: 2px solid #abc38e;
+  border-radius: 11px;
+  font-family: 'PT Mono', monospace;
+  font-size: 15px;
+  margin-top: 5px;
+  padding: 3px;
+`;
+
+const StyledListingList = styled.div`
   display: flex;
   flex-wrap: wrap;
 `;
 
-function ListingFeed({ listings, users, onFavourise, favourites }) {
+function ListingFeed({
+  listings,
+  users,
+  onFavourise,
+  favourites,
+  onTypeFilter,
+  typeFilter
+}) {
+  const [filteredListings, setFilteredListings] = useState(
+    listings.filter(
+      listing => typeFilter === 'all' || listing.type === typeFilter
```

Comment:
habe es nicht ausprobiert aber testest du ob es einen Filter gibt und falls ja, soll dieser angewandt werden?  so wie ich das lese filtert es nach: ist "all" gesetzt oder "typeFilter" und wenn eines von beiden stimmt wird gefiltert...ich frage mich ob das so korrekt/gewollt ist

Reply:
das ist so gewollt, ich habe keinen Anzeigen Typ 'all', daher wollte ich abfangen wenn der Filter all ist soll sozusagen nicht gefiltert werden. Vielleicht gibt es einen eleganteren Weg? Vielleicht stelle ich es mal in den question channel